File: test/randomtest/scallaexternalgitone/pipelines/SC_CONFIG_EXT_GIT_ONE/code/src/main/scala/io/prophecy/pipelines/sc_config_ext_git_one/graph/customers_orders.scala
package io.prophecy.pipelines.sc_config_ext_git_one.graph

import io.prophecy.libs._
import io.prophecy.pipelines.sc_config_ext_git_one.config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object customers_orders {

  def apply(spark: SparkSession): DataFrame =
    spark.read
      .format("csv")
      .option("header",                   true)
      .option("sep",                      ",")
      .option("ignoreLeadingWhiteSpace",  false)
      .option("ignoreTrailingWhiteSpace", false)
      .schema(
        StructType(
          Array(
            StructField("customer_id",         IntegerType, true),
            StructField("orders",              LongType,    false),
            StructField("amounts",             DoubleType,  true),
            StructField("account_length_days", IntegerType, true)
          )
        )
      )
      .load(
        "dbfs:/Prophecy/abhisheks+e2etests1@simpledatalabs.com/CustomersOrders.csv"
      )

}

File: test/ayush.dev/python/pipelines/pipeline_dataset_different_account_teams/code/pipeline_dataset_different_account_teams/graph/customers_orders.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pipeline_dataset_different_account_teams.config.ConfigStore import *
from pipeline_dataset_different_account_teams.udfs.UDFs import *

def customers_orders(spark: SparkSession) -> DataFrame:
    return spark.read\
        .schema(
          StructType([
            StructField("customer_id", IntegerType(), True), StructField("orders", LongType(), False), StructField("amounts", DoubleType(), True), StructField("account_length_days", IntegerType(), True)
        ])
        )\
        .option("header", True)\
        .option("sep", ",")\
        .option("ignoreLeadingWhiteSpace", True)\
        .option("ignoreTrailingWhiteSpace", True)\
        .csv("dbfs:/Prophecy/abhisheks+e2etests@prophecy.io/CustomersOrders.csv")
